### DISCUSSION TITLE
Big WOWs for v3 GA

### DISCUSSION THREAD

**Discussion by JillieBeanSim** [zowe/zowe-explorer-vscode#2391] [Squad Discussion]
I wanted to create a list of big WOWs we may want to deliver as part of v3.0.0 in 2024

1. Unix command capability
2. Read-only mode for JES spool files (requires engine update to vscode 1.79.0)
3. Properties view for z/OS Unix files
4. Option to convert v1 profiles to team config file
5. Config file creation/editing UI
6. Drag & Drop/copy & paste (local to remote & reverse)/download to local not in temp (requires engine update to vscode 1.66.0 for drag & drop APIs)

Feel free to discuss these and other items we would like to try to include as big WOWs for the v3 GA

**Comment by zFernand0** [comment]
I believe these could be another couple of big WOWs:
- https://github.com/zowe/vscode-extension-for-zowe/issues/2373
- https://github.com/zowe/vscode-extension-for-zowe/issues/2297

**Reply by JillieBeanSim** [reply]
Thanks @zFernand0 for the suggestions.
I think the cert support may need to be introduced sooner in a v2, it shouldn't be a breaking change.
Also, Yes to anything being read-only going into v3 as it requires the vsc engine to be updated to 1.79.0.

**Comment by FALLAI-Denis** [comment]
Hi,

I suggest integrating the redesign of job output management in particular to give a user experience closer to that of SDSF or SYSVIEW.

- #2255 
- #2256 
- #2257 
- #2258 
- #1597

**Reply by zFernand0** [reply]
May I suggest a bulleted list? 
That way you don't have to copy the headers (if they change in the future)
- https://github.com/zowe/vscode-extension-for-zowe/issues/2255
- https://github.com/zowe/vscode-extension-for-zowe/issues/2256
- https://github.com/zowe/vscode-extension-for-zowe/issues/2257
- https://github.com/zowe/vscode-extension-for-zowe/issues/2258
- https://github.com/zowe/vscode-extension-for-zowe/issues/1597

**Reply by JillieBeanSim** [reply]
Thanks @FALLAI-Denis, we have the epic that has these stories in our [Roadmap](https://github.com/zowe/vscode-extension-for-zowe/wiki/2023-Zowe-Explorer-Roadmap#add-more-functionality-to-the-jobs-tree-view) for targeted delivery by end of 2023, we could definitely put them in as v3 enhancements or deliver to the live 2.x work 😄

**Reply by FALLAI-Denis** [reply]
@zFernand0: I looked for a long time how to do without finding... thank you for having communicated the solution to me.
@JillieBeanSim: If we can have the functionality before v3, that's even better!

**Comment by JillieBeanSim** [comment]
We also have the list of `community-upvoted` [items](https://github.com/zowe/vscode-extension-for-zowe/issues?q=is%3Aopen+is%3Aissue+label%3Acommunity-upvoted) to choose from as well

- https://github.com/zowe/vscode-extension-for-zowe/issues/1930 (needed for CII badge work)
- https://github.com/zowe/vscode-extension-for-zowe/issues/1784
- https://github.com/zowe/vscode-extension-for-zowe/issues/1667
- https://github.com/zowe/vscode-extension-for-zowe/issues/1597 (already planned in Roadmap)
- https://github.com/zowe/vscode-extension-for-zowe/issues/1590
- https://github.com/zowe/vscode-extension-for-zowe/issues/1589
- https://github.com/zowe/vscode-extension-for-zowe/issues/1568
- https://github.com/zowe/vscode-extension-for-zowe/issues/1326 (committed in 23PI3 for v3)

**Comment by GilPen** [comment]
Hello

May I suggest to control which tiles I would like to display in the API CATALOG ?
For our example; we would like to display only our own API developements for our users and not  expose Z/OSMF services or caching services  in the swagger.

Regards
Gilles

**Reply by t1m0thyj** [reply]
@GilPen Thanks for your suggestion! This repository is for the Zowe Explorer VS Code extension. If you have an enhancement request for the Zowe API catalog, please create an issue in the repository https://github.com/zowe/api-layer/issues instead 🙂

**Reply by zFernand0** [reply]
Hey @zowe/api-mediation team,
Check out this suggestion 😋

**Reply by boris-bc** [reply]
This request is already in the APIML repo: https://github.com/zowe/api-layer/issues/2984
